src/log.c

```diff
@@ -237,15 +234,25 @@ _rotate_log_file(void)
     size_t len = strlen(log_file);
     gchar* log_file_new = malloc(len + 4);
 
-    // find an empty name. from .log -> log.01 -> log.99
-    for (int i = 1; i < 100; i++) {
-        g_sprintf(log_file_new, "%s.%02d", log_file, i);
+    // the mainlog file should always end in '.log', lets remove this last part
+    // so that we can have profanity001.log later
```

Comment:
let's use the "standard way"^TM when naming logfiles which is separated by a `.` - `profanity.NNN.log`